lib/index.ts

```diff
@@ -673,7 +673,9 @@ export default class Backburner {
 
   private _runExpiredTimers() {
     this._timerTimeoutId = null;
-    this.run(this, this._scheduleExpiredTimers);
+    this.begin();
+    this._scheduleExpiredTimers();
```

Comment:
that one was used to cleanup if it throws error, but since `_scheduleExpiredTimers` is not supposed to throw error I think it is not needed here :), what you say ?